README.md

```diff
@@ -91,6 +91,14 @@ To enable message feedback, you will need to set up CosmosDB resources. Then spe
 /.env
 - `AZURE_COSMOSDB_ENABLE_FEEDBACK=True`
 
+#### Local Setup: Enable SQL Server
+To enable SQL Server, you will need to set up SQL Server resources. Then specify these additional environment variables:
+- `DATASOURCE_TYPE`
+- `AZURE_SQL_SERVER_CONNECTION_STRING`
+- `AZURE_SQL_SERVER_TABLE_SCHEMA`
+
+NOTE: You may find you need to set: MacOS: `export NODE_OPTIONS="--max-old-space-size=8192"` or Windows: `set NODE_OPTIONS=--max-old-space-size=8192` to avoid running out of memory when building the frontend.
```

Comment:
This will be true for any scenario now, since the change is on the main branch.  Can you move this note to another section (maybe the prerequisites or a more general part of the "Local Setup" instructions)?